packages/core/src/routes-me/user-assets.ts

```diff
@@ -66,7 +66,7 @@ export default function userAssetsRoutes<T extends AuthedMeRouter>(...[router]:
 
       const file = bodyFiles[0];
       assertThat(file, 'guard.invalid_input');
-      assertThat(file.size <= maxUploadFileSize, 'guard.file_size_exceeded');
+      assertThat(file.size <= maxUploadImageSize, 'guard.file_size_exceeded');
```

Comment:
Need to discuss this. By reading the API path, I can not know that this API is for uploading users pic/avatar only, so it does not make sense to limit the file size to be the max size of image files?